doc/glm/glm_intro.rst

```diff
@@ -141,7 +141,7 @@ i.e. values of ``z`` larger than the :math:`(1-\alpha)`-quantile of the standard
 The first idea that one might think of is to take a much smaller :math:`\alpha`:
 for instance, if we take, :math:`\alpha=\frac{0.05}{n\_voxels}`
 then the expected number of false discoveries is only about 0.05, meaning
-that there is a 5% chance that a truly inactive voxel is declared active.
+that there is a 5% chance that a truly inactive :term:`voxel` is declared active.
```

Comment:
Unsure whether we really need to track all occurrences of 'voxels' ;-)

Reply:
Trust me: I left some out. 😉